src/openrct2/scripting/ScPark.hpp

```diff
@@ -30,12 +30,16 @@ namespace OpenRCT2::Scripting
         "attraction", "peep_on_attraction", "peep", "money", "blank", "research", "guests", "award", "chart",
     };
 
-    inline uint8_t GetParkMessageType(const std::string& key)
+    inline News::ItemType GetParkMessageType(const std::string& key)
     {
-        auto it = std::find(std::begin(ParkMessageTypeStrings), std::end(ParkMessageTypeStrings), key);
-        return it != std::end(ParkMessageTypeStrings)
-            ? static_cast<uint8_t>(NEWS_ITEM_RIDE + std::distance(std::begin(ParkMessageTypeStrings), it))
-            : static_cast<uint8_t>(NEWS_ITEM_BLANK);
+        // Get the first ItemType that appears in ParkMessageTypestrings
+        auto firstType = static_cast<uint8_t>(News::ItemType::Ride);
+
+        auto begin = std::begin(ParkMessageTypeStrings);
+        auto end = std::end(ParkMessageTypeStrings);
+
+        auto it = std::find(begin, end, key);
+        return it != end ? static_cast<News::ItemType>(firstType + std::distance(begin, it)) : News::ItemType::Blank;
     }
 
     inline std::string GetParkMessageType(uint8_t type)
```

Comment:
Make this take the specific type then static cast to `uint8_t` when required in the function.